.gitmodules

```diff
@@ -34,3 +31,6 @@
 [submodule "3rdparty/filesystem"]
 	path = 3rdparty/filesystem
 	url = https://github.com/gulrak/filesystem.git
+[submodule "3rdparty/fiber"]
+	path = 3rdparty/fiber
+	url = https://github.com/bbannier/fiber.git
```

Comment:
Change this to upstream if https://github.com/simonfxr/fiber/pull/1 gets merged, else create and use zeek fork.